frontend/src/pages/learn/index.tsx

```diff
@@ -188,11 +190,11 @@ const Learn: FC = () => {
         ]}
       >
         <HeroBanner imageProps={{ className: 'md:object-right-bottom', src: learnBannerImage }}>
-          <h1 className="mb-2 font-display text-4xl font-bold text-primary-700 md:mb-4 md:text-6xl">
+        <h1 className={`mb-2 font-display text-4xl font-bold text-primary-700 md:mb-4 ${locale === 'en' ? 'md:text-6xl' : 'md:text-4xl lg:text-6xl'}`}>
```

Comment:
It should be `md:text-4xl lg:text-6xl` for both languages.

Reply:
Fixed